source/frontend/src/components/common/mapFSM/MapStateMachineContext.tsx

```diff
@@ -121,15 +124,21 @@ export const MapStateMachineProvider: React.FC<React.PropsWithChildren<unknown>>
     actions: {
       navigateToProperty: context => {
         const selectedFeatureData = context.mapLocationFeatureDataset;
-        if (exists(selectedFeatureData?.pimsFeature?.properties?.PROPERTY_ID)) {
-          const pimsFeature = selectedFeatureData.pimsFeature;
+
+        // if there is more that one property on the location, further action is needed.
+        if (selectedFeatureData.parcelFeatures?.length > 1) {
```

Comment:
what about multiple pims features?

Reply:
I think the current behaviour will suffice for getting the first one. In general we need a better way to interact with the map.